Server/ConcordiaCurriculumManager/Controllers/CourseController.cs

```diff
@@ -70,12 +69,12 @@ public async Task<ActionResult> GetAllCourseSettings()
     [SwaggerResponse(StatusCodes.Status500InternalServerError, "Unexpected error")]
     [SwaggerResponse(StatusCodes.Status201Created, "Course creation dossier created successfully", typeof(Guid))]
     [SwaggerRequestExample(typeof(CourseCreationInitiationDTO), typeof(CourseCreationInitiationDTOExample))]
-    public async Task<ActionResult> InitiateCourseCreation([FromBody, Required] CourseCreationInitiationDTO initiation)
+    public async Task<ActionResult> InitiateCourseCreation([FromBody, Required] CourseCreationInitiationDTO initiation, Guid dossierId)
     {
         try
         {
             var user = await _userService.GetCurrentUser();
-            var response = await _courseService.InitiateCourseCreation(initiation, user);
+            var response = await _courseService.InitiateCourseCreation(initiation, user, dossierId);
 
             return Created($"/{nameof(InitiateCourseCreation)}", response.Id);
```

Comment:
This is more an issue that's my fault that I forgot to mention. We should have a new class CourseCreationRequestDTO that we map data to with automapper (just the important fields) and return that instead of just an ID. You can see examples in other controllers of mapping to a DTO, or ask me if you're not sure.

Reply:
I figured out how do it. I pushed the new changes. Let me know if I should change something else.